pages/playground.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useEffect } from "react";
+import TextArea from "components/TextArea";
+import Button from "components/Button";
+import styles from "styles/modules/Playground.module.scss";
+import { useTheme } from "providers/theme";
+import { showError } from "utils/toast-utils";
+import ResourceSearchAndResults from "components/playground/ResourceSearchAndResults";
+import PolicySearchAndResults from "components/playground/PolicySearchAndResults";
+import EvaluationResult from "components/playground/EvaluationResult";
+import { usePolicies } from "providers/policies";
+import { policyActions } from "reducers/policies";
+import { useResources } from "providers/resources";
+import { resourceActions } from "reducers/resources";
+
+const PolicyEvaluationPlayground = () => {
+  const { theme } = useTheme();
+  const { state, dispatch: policyDispatch } = usePolicies();
+  const { dispatch: resourceDispatch } = useResources();
+
+  const [evaluationResults, setEvaluationResults] = useState(null);
+  const [evaluationLoading, setEvaluationLoading] = useState(false);
+
+  const evaluatePolicy = async (event) => {
+    event.preventDefault();
+    const requestBody = {
+      resourceUri: state.evaluationResource.uri,
```

Comment:
This probably shouldn't be working... I think the key is `resourceURI`.

Reply:
I gave this some thought last night and remembered that grafeas uses resourceUri and not resourceURI. I think (not sure) general standards support the ladder, but we probably want to be consistent with the grafeas api. So no need to make a change from your end

Reply:
It is `uri` because of this [mapping happening on the backend](https://github.com/rode/rode-ui/blob/5a3cb391a7fe78002ca424d493a9709f7e909112/pages/api/resources.js#L50-L53).  I could definitely change it to be consistent with Rode if we care strongly about it!